### PR TITLE
Add file2.rb

### PR DESCRIPTION
**Pull request by tavisca-tkatyal**
@tavisca-vtripathi

### CODE REVIEW

File: lib/file2.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env ruby
+require 'find'
+require 'pathname'
```
Comment (tavisca-vtripathi) [inline lib/file2.rb:3]: Put the logic in a method

**Review by tavisca-vtripathi — CHANGES_REQUESTED** [review]

**Comment by tavisca-tkatyal** [comment]
@tavisca-vtripathi 
method add in file2.rb

File: lib/file2.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+puts "Hello user, Please Input the folder name"
```
Comment (tavisca-vtripathi) [inline lib/file2.rb]: Put user input as well in a method and do the calling.

**Review by tavisca-vtripathi — CHANGES_REQUESTED** [review]

**Comment by tavisca-tkatyal** [comment]
done the suggested Changes.
@tavisca-vtripathi

**Review by tavisca-vtripathi — APPROVED** [review]

**Pull request closed without merging**